CHANGELOG.md

```diff
@@ -551,7 +551,7 @@ current changes on git with [previous release tags][git_tag_comparison].
 - [Spotlights][4715]
 - [Camera Driven Rendering][4745]
 - [Camera Driven Viewports][4898]
-- [Visibilty Inheritance, universal `ComputedVisibility`, and `RenderLayers` support][5310]
+- [Visibility Inheritance, universal `ComputedVisibility`, and `RenderLayers` support][5310]
```

Comment:
These are directly reflective of the PRs that made them, unless we rename them. These should not be touched.

Reply:
Okay, I rollbacked them. 😄